src/xo-app/dashboard/overview/index.js

```diff
@@ -61,7 +72,131 @@ class PatchesCard extends Component {
   }
 }
 
-// ===================================================================
+@propTypes({
+  resourceSet: propTypes.object.isRequired
+})
+@injectIntl
+class ResourceSetCard extends Component {
+  _getQuotas = createSelector(
+    () => this.props.resourceSet.limits,
+    limits => {
+      const quotas = {}
+
+      forEach(resources, resource => {
+        if (limits[resource] != null) {
+          const {
+            available,
+            total
+          } = limits[resource]
+
+          quotas[resource] = {
+            available,
+            total,
+            usage: total - available
+          }
+        } else {
+          quotas[resource] = {
+            available: 0,
+            total: 0,
+            usage: 0
+          }
+        }
+      })
+
+      return quotas
+    }
+  )
+
+  render () {
+    const {
+      cpus,
+      disk,
+      memory
+    } = this._getQuotas()
+    const { formatMessage } = this.props.intl
+
+    return <Card>
+      <CardHeader>
+        <Icon icon='menu-self-service' /> {this.props.resourceSet.name}
+      </CardHeader>
+      <CardBlock>
+        <Container>
+          <Row>
+            <Col mediumSize={4}>
+              <Card>
+                <CardHeader>
+                  <Icon icon='cpu' /> {_('cpuStatePanel')}
+                </CardHeader>
+                <CardBlock>
+                  <ChartistGraph
+                    data={{
+                      labels: [ formatMessage(messages.availableResourceLabel), formatMessage(messages.usedResourceLabel) ],
```

Comment:
You could create a `const labels = [ formatMessage(messages.availableResourceLabel), formatMessage(messages.usedResourceLabel) ]` to simplify these.